plugin.js

```diff
@@ -214,6 +214,8 @@ module.exports = fastifyCookie
 module.exports.default = fastifyCookie // supersedes fastifyCookie.default = fastifyCookie
 module.exports.fastifyCookie = fastifyCookie // supersedes fastifyCookie.fastifyCookie = fastifyCookie
 
+module.exports.cookie = cookie
```

Comment:
Can you change our test to load cookie from the top level module then?

Reply:
idk